BeagleBone_black/src/sensors/vl6180.hpp

```diff
@@ -84,8 +103,11 @@ class Vl6180: public Thread {
     *  @return int Returns 0 if successful
     */
   int writeByte(uint16_t reg_add, char data);
+  Logger& log_;
   bool on_;
   bool continuous_mode_;
+  uint8_t i2c_addr_ = DEFAULT_I2C_SLAVE_ADDR;
+  I2C& i2c_ = I2C::getInstance();
```

Comment:
consider assigning these default values in the constructor

Reply:
I have assigned the i2c_addr_ in the constructor although the i2c_ instantiation is giving me an error when I assign it in the constructor.

Reply:
That worked! Thanks.